main.tf

```diff
@@ -55,10 +55,19 @@ resource "google_bigquery_table" "main" {
   friendly_name   = each.key
   table_id        = each.key
   labels          = each.value["labels"]
-  schema          = file(each.value["schema"])
+  schema          = each.value["schema"] != null ? file(each.value["schema"]) : null
```

Comment:
Lookup will not work since when I feed it to file(), it raises null exception. 

Asking users to provide file directly instead of name is a cleaner option, then we can use lookup. But it requires backward incompatible change similar to view. 

What do you think, should I go forward?

Reply:
@umairidris good point, we thought already about it, but I'm afraid not related to that PR.

Reply:
Understood, making an unncessary breaking change here would be counter productive. If we decide to make a new release then making the change can be worthwhile. Thanks for explaining.